packages/plugin-page-view-tracking/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@amplitude/plugin-page-view-tracking",
```

Comment:
Unless this is supported by all platforms I think we should include a postfix to be explicit `@amplitude/plugin-page-view-tracking-browser`

Reply:
Sure. Let's call `@amplitude/plugin-page-view-tracking-browser`